### PR TITLE
MPP-3243 Rebrand Firefox Account in Firefox Relay

### PR DESCRIPTION
**Pull request by lloan**
This PR fixes MPP-3243

L10N: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/158

# New feature description

As a account customer on Firefox Relay, seeing consistent branding of accounts across Mozilla surfaces helps me trust the authentication experience, and not feel confused about the services I use. 

Rewrite Firefox Account(s) to Mozilla Account(s)

Comments, documentation updated - not behind a flag. 

# Screenshot (if applicable)

image

image

# How to test

Any and all instances of Firefox Account(s) should be rewritten to Mozilla Account(s)

User facing strings are behind a flag -> `firefox_account_rebrand`

# Checklist (Definition of Done)
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] I've added or updated relevant docs in the docs/ directory
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] l10n changes have been submitted to the l10n repository, if any.

### CODE REVIEW

**Review by stephaniehobson — APPROVED** [review]
✅ Acknowledging we are also doing these updates on the migrated site.

File: frontend/pendingTranslations.ftl [hunk 1]
```diff
@@ -35,3 +35,12 @@ profile-label-set-your-custom-domain-free-user = Get your own email domain with
 
 tooltip-email-domain-explanation-title-free = Get your own { -brand-name-relay } email domain
 tooltip-email-domain-explanation-part-one-free = With { -brand-name-premium }, you can create unlimited, custom { -brand-name-relay } masks on the go using a unique { -brand-name-relay } email domain — you won’t even have to generate them here first.
+
+nav-profile-manage-fxa-v2 = Manage your { -brand-name-mozilla-account(capitalization: "uppercase") }
+nav-profile-image-alt-v2 = { -brand-name-mozilla-account(capitalization: "uppercase") } Avatar
+faq-question-acceptable-use-answer-measure-account-v2 = Requiring a { -brand-name-mozilla-account(capitalization: "uppercase") } with a verified email address
```
Comment (bcolsson) [inline frontend/pendingTranslations.ftl:41]: These should probably all be lowercase to be consistent with the [Mozilla account content guidelines](https://www.figma.com/file/7nsOBSt3VTeCimDqavQT9J/Mozilla-Accounts-Content-Guidelines?type=design&node-id=1903-403&mode=design&t=gtC3OzSNcm5JK5b5-0)

Comment (lloan) [reply frontend/pendingTranslations.ftl:41]: We actually removed 'mozilla/firefox' and doing 'account' based on the guidelines. Updated L10N strings to reflect that (separate PRs) - updating this one shortly.

**Comment by lloan** [comment]
Removed flag that was originally requested. Removed 'Mozilla Accounts' from customer facing strings - now its just account(s).

File: frontend/pendingTranslations.ftl [hunk 2]
```diff
@@ -35,3 +35,7 @@ profile-label-set-your-custom-domain-free-user = Get your own email domain with
 
 tooltip-email-domain-explanation-title-free = Get your own { -brand-name-relay } email domain
 tooltip-email-domain-explanation-part-one-free = With { -brand-name-premium }, you can create unlimited, custom { -brand-name-relay } masks on the go using a unique { -brand-name-relay } email domain — you won’t even have to generate them here first.
+
+nav-profile-manage-fxa-v2 = Manage your account
```
Comment (groovecoder) [inline frontend/pendingTranslations.ftl:39]: suggestion (non-blocking): Should we drop `fxa` from this string ID too?
```suggestion
nav-profile-manage-account = Manage your account
```

File: frontend/pendingTranslations.ftl [hunk 3]
```diff
@@ -35,3 +35,7 @@ profile-label-set-your-custom-domain-free-user = Get your own email domain with
...
+nav-profile-image-alt-v2 = account avatar
```
[the remark this answers is not in the document]
Comment (lloan) [reply frontend/pendingTranslations.ftl:40]: From the L10N PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/158#pullrequestreview-1648815438

File: frontend/pendingTranslations.ftl (hunk 2 above)
Comment (lloan) [reply frontend/pendingTranslations.ftl]: done

**Review by groovecoder — APPROVED** [review]
Thanks for the updates!

**Pull request merged**